.circleci/config.yml

```diff
@@ -68,20 +68,67 @@ commands:
 # Define a job to be invoked later in a workflow.
 # See: https://circleci.com/docs/2.0/configuration-reference/#jobs
 jobs:
-  build-and-test:
+  init:
     docker:
       - image: sematicai/sematic-ci:latest
     steps:
       - checkout
       - install-ci-pip-deps
+      - persist_to_workspace:
+          root: /home/circleci
+          paths:
+            - .pyenv
+            - project
+
+  build-and-lint:
+    docker:
+      - image: sematicai/sematic-ci:latest
+    steps:
+      - attach_workspace:
+          at: /home/circleci
       - do-static-analysis
+      
+  unit-tests:
+    docker:
+      - image: sematicai/sematic-ci:latest
+    steps:
+      - attach_workspace:
+          at: /home/circleci
       - non-coverage-tests
+
+  integration-test:
+    docker:
+      - image: sematicai/sematic-ci:latest
+    steps:
+      - attach_workspace:
+          at: /home/circleci
       - installation-tests
+      
+  finalize:
+    docker:
+      - image: cimg/base:2023.01
```

Comment:
Why not use the same image?

Reply:
This image could load more quickly. Judging by the workload of the job (slack notify only), there seems no need to use our own heavier image.